medusa/failed_processor.py

```diff
@@ -36,39 +39,41 @@ def __init__(self, dirName, nzbName):
 
     def process(self):
         """
-        Do the actual work
+        Do the actual work.
 
         :return: True
         """
         self._log(u'Failed download detected: ({nzb}, {dir})'.format(nzb=self.nzb_name, dir=self.dir_name))
 
-        releaseName = show_name_helpers.determineReleaseName(self.dir_name, self.nzb_name)
-        if not releaseName:
-            self._log(u'Warning: unable to find a valid release name.', logger.WARNING)
-            raise FailedPostProcessingFailedException()
-
         try:
-            parsed = NameParser().parse(releaseName)
+            parsed = NameParser().parse(self.nzb_name or self.dir_name)
```

Comment:
@medariox this is what Im concern

Dir name is NOT the nzb name right? What can I add here that will always be the nzb folder name?